src/functional/ffront/type_info.py

```diff
@@ -171,6 +171,27 @@ def is_integral(symbol_type: ct.SymbolType) -> bool:
     ]
 
 
+def is_number(symbol_type: ct.SymbolType) -> bool:
+    """
+    Check if ``symbol_type`` is either intergral or float.
+
+    Examples:
+    ---------
+    >>> is_number(ct.ScalarType(kind=ct.ScalarKind.FLOAT64))
+    True
+    >>> is_number(ct.ScalarType(kind=ct.ScalarKind.INT32))
+    True
+    >>> is_number(ct.ScalarType(kind=ct.ScalarKind.BOOL))
+    False
+    >>> is_number(ct.FieldType(dims=[], dtype=ct.ScalarType(kind=ct.ScalarKind.INT)))
+    False
+    """
+    if not isinstance(symbol_type, ct.ScalarType):
+        return False
+    # TODO: @nfarabullini re-factor is_arithmetic such it only checks for scalars and the emtpy field pass is in an another function
```

Comment:
Line length. You can configure PyCharm to do that for you (Ctrl + Alt + L on Linux). The length is configured in Settings > Editor > Code Style > Hard wrap at.